index.html

```diff
@@ -25,7 +25,17 @@
 
   <!-- Classes Section -->
   <section id="classes">
-
+      <h2>Classes</h2>
+    <ul>
+      <li><img src="./assets/html.svg" alt=""></li>
+      <li><img src="./assets/css.svg" alt=""></li>
+      <li><img src="./assets/javascript.svg" alt=""></li>
+      <li><img src="./assets/nodejs.svg" alt=""></li>
+      <li><img src="./assets/php.svg" alt=""></li>
+      <li><img src="./assets/python.svg" alt=""></li>
+      <li><img src="./assets/sql.svg" alt=""></li>
+      <li><img src="./assets/b.svg" alt=""></li>
```

Comment:
Don't leave empty alt tags. Please delete them or add some reasonable names to them.